apps/web/src/app/(dashboard)/documents/[id]/page.tsx

```diff
@@ -11,7 +11,7 @@ import { DocumentStatus as InternalDocumentStatus } from '@documenso/prisma/clie
 import { LazyPDFViewer } from '@documenso/ui/primitives/lazy-pdf-viewer';
 
 import { EditDocumentForm } from '~/app/(dashboard)/documents/[id]/edit-document';
-import { StackAvatarsWithTooltip } from '~/components/(dashboard)/avatar/stack-avatars-with-tooltip';
+import { StackAvatarsWithUI } from '~/components/(dashboard)/avatar/stack-avatars-with-ui';
```

Comment:
I think StackAvatars or StackAvatarsUI would be better. Wdyt?

Reply:
I've changed the component name to StackAvatarsUI. :)